x/treasury/client/cli/query.go

```diff
@@ -265,45 +154,23 @@ $ terracli query treasury tax-proceeds 14
 		},
 	}
 
-	cmd.Flags().String(flagEpoch, "", "(optional) an epoch number which you wants to get tax proceeds of; default is current epoch")
-
 	return cmd
 }
 
 // GetCmdQuerySeigniorageProceeds implements the query seigniorage-proceeds command.
 func GetCmdQuerySeigniorageProceeds(cdc *codec.Codec) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "seigniorage-proceeds [epoch]",
```

Comment:
Same comment as `tax-proceeds`. Seems like giving users the ability to query "seigniorage proceeds" when it just imputes the amount of luna burned is not necessary

Reply:
FCD will use this

Reply:
Ok.